RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
+snapshot version (see "Prepare For The Next Release" below).
+
+Note that, in the example above, the docker image pushed to DockerHub will not
+be tagged with "latest" but will instead be tagged with "1.0.0.Final."
+
+== Notify Istio and Openshift Istio Teams for changes
+
+There are no fixed steps for releasing a new version on the istio installer,
+each release is kind of different.
+
+This has been done a few times before, so they can be used as references:
+
+* https://github.com/istio/istio/pull/5869
+* https://github.com/istio/istio/pull/5921
+* https://github.com/istio/istio/pull/6192
+* https://github.com/istio/istio/pull/6592
+* https://github.com/istio/istio/pull/6594
+* https://github.com/istio/istio/pull/7042
+* https://github.com/istio/istio/pull/7007
+
+Those PRs cover both the ansible-installer and the helm installer updates.
+
+== Updating JIRA issues
 
-Once the tag has been pushed, Travis will immediately run another build, but this time will tag the docker image with the same version string as the git tag name. In the example above, the docker image pushed to DockerHub will not be tagged with "latest" but will instead be tagged with "v1.0.0".
+Here, we move all issues tagged as `Ready to Release` to `Released`, with
+`Fix-Version` set to the actual release version.
 
-You can let the committers know that the release has been completed and PRs can begin to be merged again. Though, it would be best to wait for this notification to go out until after you bump up the version string to the next snapshot version (see "Prepare For The Next Release" below).
+After that, close the release issues, and then, create a new `Release` in JIRA
+for the next release.
 
-At this point, Travis will run the test suite, build the binary, and push the versioned docker image to link:https://hub.docker.com/r/kiali/kiali[DockerHub]. In the example above, the docker image pushed to DockerHub will not be tagged with "latest" but will instead be tagged with "1.0.0.Final."
+== Send out notifications
 
-== Prepare For The Next Release
+Finally, we just need to send the nofications to create awareness of the new
+release.
+
+Post an email about the release on our
+link:https://groups.google.com/forum/#!search/kiali-dev[Mailing List].
+
+Send a message on our channel at Freenode, #kiali.
+
+Post a message on our twitter,
+link:https://www.twitter.com/KialiProject[@KialiProject]. If you don't know
+who's responsible for the account, ask on IRC.
+
+== Preparing for the Next Release
+
+After the release happens, you'll need to prepare Kiali for the next one. This means changing the Makefile to the defaults:
+
+[source, diff]
+----
+diff --git a/Makefile b/Makefile
+index c1f7f80..4a8af40 100644
+--- a/Makefile
++++ b/Makefile
+@@ -3,7 +3,7 @@ SHELL=/bin/bash
+ 
+ # Identifies the current build.
+ # These will be embedded in the app and displayed when it starts.
+-VERSION ?= v1.0.0
++VERSION ?= v1.1.0
```

Comment:
Do we want to add a suffix to the new development version as we've been doing, such as "-SNAPSHOT" so the version is `v1.1.0-SNAPSHOT` in this example. Anyone getting images showing this SNAPSHOT version in the About box and logs will see it is obviously a development build (whereas if someone see v1.1.0, it won't be obvious that this isn't a released image).

Reply:
+1 and I was wondering the same about the UI https://github.com/kiali/kiali-ui/pull/516